test/specs/element.point.tests.js

```diff
@@ -222,7 +222,7 @@ describe('Point element tests', function() {
 			15 - offset,
 			Math.SQRT2 * 2,
 			Math.SQRT2 * 2,
-			2 / 2
+			2 * 0.425
```

Comment:
maybe a comment here explaining what `0.425` is would be helpful

Reply:
`2 / 2` wasn't explained, you need to read the `roundedRect` method to understand why and that's the same for `0.425`, which one is [properly explained in the method body](/chartjs/Chart.js/pull/5597/files#diff-5a99d0ca9a04b73627a387ff34c96da2R95). I don't think we should duplicate the comment here since it's exactly the same value in the calling code.